python/examples/core/harness.py

```diff
@@ -465,8 +541,12 @@ def test_harness(problem_factory: Callable[
                 'pytorch_benchmark'], n, args, problem_sizes_dict, np_types),
             gflops, gbytes, n_iters)
 
-        measurements.append('pytorch', np_types,
-                            compile_time_problem_sizes_dict,
+        measurements.append('pytorch', np_types, runtime_only_sizes,
```

Comment:
that nesting of numpy and pytorch within the execution was really to get started and likely needs to be torn out and refactored (not in this PR).